py3status/modules/window_title.py

```diff
@@ -51,12 +51,12 @@ def __init__(self):
     def window_title(self):
         window = find_focused(i3.get_tree())
 
+        if window['name'] is None:
```

Comment:
I think we need to do 
```
if not window or window.get('name') is None:
    window = {'name': ''}
```

later we have `if window and 'name' in window and window['name']...` so looks like window['name']` or even window can be missing.

Also we can just remove `if window and 'name' in window and window['name']...` test and all the `transformed` bits as they are not needed/used